Nodejs/Product/Nodejs/NodejsProject.cs

```diff
@@ -1,1011 +1,1014 @@
+                if (String.Equals(ext, ".js", StringComparison.OrdinalIgnoreCase)) {
+                    return true;
+                }
+            }
+            return false;
+        }
+
+        int IOleCommandTarget.QueryStatus(ref Guid pguidCmdGroup, uint cCmds, OLECMD[] prgCmds, IntPtr pCmdText) {
+            if (pguidCmdGroup == Guids.Eureka) {
+                for (int i = 0; i < prgCmds.Length; i++) {
+                    switch (prgCmds[i].cmdID) {
+                        case 0x102: // View in Web Page Inspector from Eureka web tools
+                            prgCmds[i].cmdf = (uint)(OLECMDF.OLECMDF_INVISIBLE | OLECMDF.OLECMDF_SUPPORTED | OLECMDF.OLECMDF_ENABLED);
+                            return VSConstants.S_OK;
+                    }
+                }
+            } else if (pguidCmdGroup == Guids.VenusCommandId) {
+                for (int i = 0; i < prgCmds.Length; i++) {
+                    switch (prgCmds[i].cmdID) {
+                        case 0x034: /* add app assembly folder */
+                        case 0x035: /* add app code folder */
+                        case 0x036: /* add global resources */
+                        case 0x037: /* add local resources */
+                        case 0x038: /* add web refs folder */
+                        case 0x039: /* add data folder */
+                        case 0x040: /* add browser folders */
+                        case 0x041: /* theme */
+                        case 0x054: /* package settings */
+                        case 0x055: /* context package settings */
+
+                            prgCmds[i].cmdf = (uint)(OLECMDF.OLECMDF_INVISIBLE | OLECMDF.OLECMDF_SUPPORTED | OLECMDF.OLECMDF_ENABLED);
+                            return VSConstants.S_OK;
+                    }
+                }
+            } else if (pguidCmdGroup == Guids.WebPackageCommandId) {
+                if (prgCmds[0].cmdID == 0x101 /*  EnablePublishToWindowsAzureMenuItem*/) {
+                }
+            } else if (pguidCmdGroup == Guids.WebAppCmdId) {
+                for (int i = 0; i < prgCmds.Length; i++) {
+                    switch (prgCmds[i].cmdID) {
+                        case 0x06A: /* check accessibility */
+                            prgCmds[i].cmdf = (uint)(OLECMDF.OLECMDF_INVISIBLE | OLECMDF.OLECMDF_SUPPORTED | OLECMDF.OLECMDF_DEFHIDEONCTXTMENU | OLECMDF.OLECMDF_ENABLED);
+                            return VSConstants.S_OK;
+                    }
+                }
+            } else if (pguidCmdGroup == VSConstants.VSStd2K) {
+                for (int i = 0; i < prgCmds.Length; i++) {
+                    switch ((VSConstants.VSStd2KCmdID)prgCmds[i].cmdID) {
+                        case VSConstants.VSStd2KCmdID.SETASSTARTPAGE:
+                        case VSConstants.VSStd2KCmdID.CHECK_ACCESSIBILITY:
+                            prgCmds[i].cmdf = (uint)(OLECMDF.OLECMDF_INVISIBLE | OLECMDF.OLECMDF_SUPPORTED | OLECMDF.OLECMDF_DEFHIDEONCTXTMENU | OLECMDF.
```

Comment:
Additionally you'll won't want to display the "node" and "browser" labels on folders when ES6IntelliSensePreview is enabled since it doesn't actually change anything. Disabling the option is  a good choice as well. 
https://github.com/Microsoft/nodejstools/blob/53c962502871176172ecdd2e489c715893128fa2/Nodejs/Product/Nodejs/Options/NodejsGeneralOptionsPage.cs#L89